corehq/apps/domain/models.py

```diff
@@ -594,7 +603,8 @@ def readable_languages(self):
         return ', '.join(lang_lookup[lang] or lang for lang in self.languages())
 
     @classmethod
-    @quickcache(['name'], skip_arg='strict', timeout=30*60)
+    @quickcache(['name'], skip_arg='strict', timeout=30*60,
+        memoize_timeout=10, session_function=icds_conditional_session_key())
```

Comment:
the default memoize_timeout is already 10s so I don't think you need to specify it again

Reply:
Sure